### PR TITLE
EPMRPP-39299 Fix login page elements everflow

### PR DESCRIPTION
**Pull request by AmsterGet**
image

### CODE REVIEW

**Comment by AmsterGet** [comment]
The issue with fail builds was associated with compiled files usage for tests purposes.

File: Dockerfile-full [hunk 1]
```diff
@@ -12,7 +12,7 @@ WORKDIR /usr/src/app
 COPY ./src/main/resources/public/package.json /usr/src/app/
 RUN npm install
 COPY ./src/main/resources/public/ /usr/src/app/
-RUN npm run test && npm run build
+RUN npm run build && npm run test
```
Comment (DiscoElevator) [inline Dockerfile-full]: Doesn't look like a relevant change.

File: Makefile [hunk 2]
```diff
@@ -50,8 +50,8 @@ build-server: checkstyle test
 # Builds the project
 build-statics:
 	npm --prefix $(UI_BUILD_ROOT) install
-	npm --prefix $(UI_BUILD_ROOT) run test
```
Comment (DiscoElevator) [inline Makefile]: Doesn't look like a relevant change.

File: Dockerfile-full (hunk 1 above)
Comment (AmsterGet) [reply Dockerfile-full]: Please see the comment https://github.com/reportportal/service-ui/pull/1982#issuecomment-530579950

**Pull request merged**